util/testsuite/testsuite.go

```diff
@@ -712,6 +713,47 @@ func (ts *TestSuite) GenerateTestTransferTx(options ...func(*TransactionMaker))
 	return trx
 }
 
+// GenerateTestBatchTransferTx generate a batch transfer transaction for test.
+func (ts *TestSuite) GenerateTestBatchTransferTx(recipients []payload.BatchRecipient,
```

Comment:
No need to pass `recipients` here, 
Create a randon number between (2-8) and recipient count and  create random address and values.